build/common/installer/scripts/tomlparser.rb

```diff
@@ -116,6 +138,20 @@ def populateSettingValuesFromConfigMap(parsedConfig)
             end
           end
         end
+
+        if @collectStderrLogs && !stderrSystemPods.nil? && stderrSystemPods.kind_of?(Array)
+          # Checking only for the first element to be string because toml enforces the arrays to contain elements of same type
+          if stderrSystemPods.length > 0 && stderrSystemPods[0].kind_of?(String)
+            @stderrIncludeSystemPods = stderrSystemPods.join(",")
+            puts "config::Using config map setting for stderr log collection to include system pods"
+          end
+        end
+
+        # If we have to collect logs from system pods belonging to exlcuded namespaces from either of the streams we need to reset exclude path to noop
+        if !@stdoutIncludeSystemPods.empty? || !@stderrIncludeSystemPods.empty?
+          @excludePath = "*.csv2"
```

Comment:
are we resetting excludePath if customer excluded non system namespaces (in both stderr & stdout) and using the include_system_pods? if yes, what's the behavior if the non-system excluded namespaces? are we using the pod filter in the fluent-bit Exclude_Path?   Did we test all the possible scenarios for any regressions?

Reply:
at present if namespace was present in both stderr and stdout we added it to exclude path. we can no longer do this if we want to collect logs from pods in such namespaces. therefore I reset excludePath. 

 for non-system namespaces in this scenario the records will get dropped in oms.go. unless we input both system namespace name and pod name in configmap it will be hard to avoid this